docs/source/installation/translations.rst

```diff
@@ -111,36 +111,3 @@ So if by any reason it doesn't work for you, feel free to :ref:`ask us <contact>
 
 .. |zh_Hant_TW| replace:: ``zh_Hant_TW``
 .. _zh_Hant_TW: https://www.localeplanet.com/icu/zh-Hant-TW/index.html
-
-
-File Organisation
```

Comment:
Now that we have a single POT file, I think we can simply remove this whole section